src/resources/sorting_network_master/SortingNetworkFetcher.java

```diff
@@ -0,0 +1,42 @@
+package resources.sorting_network_master;
```

Comment:
Feels kinda weird to put this file in the same directory as the added sorting network stuff?

Reply:
It matches the rest of the resources subdirectories.

Reply:
ah, true